specs/signing-scheme.md

```diff
@@ -49,7 +49,7 @@ This attribute dictates the rest of signature schema - the set of signed and uns
 Both `notary.x509` and `notary.x509.signingAuthority` signing schemes use the similar signature schema (set of signed and unsigned attributes) with the following differences.
 
 * `notary.x509` MUST use a countersignature from trusted source to determine authentic signing time (timestamp).
-This is supported through the the Timestamp signature unsigned attribute in the signature envelope. Currently Notary V2 uses a [RFC3161](ietf-rfc3161) compliant TSA signature for this purpose.
+This is supported through the the Timestamp signature unsigned attribute in the signature envelope. Currently Notary Project uses a [RFC3161](ietf-rfc3161) compliant TSA signature for this purpose.
```

Comment:
... Currently the Notary Project uses a [RFC3161](ietf-rfc3161) compliant TSA signature for this purpose.